packages/pwa-buildpack/lib/Utilities/fetcherUtils.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('node-fetch');
+
+const fetchQuery = query =>
+    fetch(new URL('graphql', process.env.MAGENTO_BACKEND_URL).toString(), {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
```

Comment:
We should also send a `Accept-Encoding: gzip` header in accordance with the [GraphQL best practices](https://graphql.org/learn/best-practices/#json-with-gzip).